wpilib/wpilib/serialport.py

```diff
@@ -201,12 +201,14 @@ def read(self, count: int) -> bytes:
         """
         return hal.readSerial(self.port, count)
 
-    def write(self, buffer: bytes) -> int:
+    def write(self, buffer: bytes, count: int) -> int:
```

Comment:
This is python, we don't need a count.

Reply:
I was questioning the java, but now I realize the JNI function needs the count

Reply:
Feel free to add a 'python-specific' note here so future reviewers aren't confused.